pkg/scheduler/backend/queue/backoff_queue_test.go

```diff
@@ -93,30 +140,30 @@ func TestBackoffQueue_popAllBackoffCompleted(t *testing.T) {
 				Pod: st.MakePod().Name("pod0").Obj(),
 			},
 			Timestamp:            fakeClock.Now().Add(-2 * time.Second),
-			Attempts:             1,
+			UnschedulableCount:   1,
 			UnschedulablePlugins: sets.New("plugin"),
 		},
 		"pod1": {
 			PodInfo: &framework.PodInfo{
 				Pod: st.MakePod().Name("pod1").Obj(),
 			},
 			Timestamp:            fakeClock.Now().Add(time.Second),
-			Attempts:             1,
+			UnschedulableCount:   1,
 			UnschedulablePlugins: sets.New("plugin"),
 		},
 		"pod2": {
 			PodInfo: &framework.PodInfo{
 				Pod: st.MakePod().Name("pod2").Obj(),
 			},
-			Timestamp: fakeClock.Now().Add(-2 * time.Second),
-			Attempts:  1,
+			Timestamp:          fakeClock.Now().Add(-2 * time.Second),
+			UnschedulableCount: 1,
```

Comment:
Probably worth change to `ConsecutiveErrorsCount` for pods that will be in `errorBackoffQ`
```suggestion
			ConsecutiveErrorsCount: 1,
```